packages/desktop-client/src/components/Titlebar.tsx

```diff
@@ -442,8 +441,8 @@ export default function Titlebar({ style }) {
       </Routes>
       <View style={{ flex: 1 }} />
       <UncategorizedButton />
-      {themesFlag && <ThemeSelector />}
```

Comment:
cc @carkom for approval on this

Reply:
oh never mind.. I thought you were removing the button entirely, but its just some small refactors.

Reply:
Yeah, just moved the flag checking to match the other buttons